cli/utilities/utils.py

```diff
@@ -800,3 +800,22 @@ def copy_ceph_sshkey_to_host(installer, node, user="root", key=CEPH_PUB_KEY):
     """
     # Copy ceph key to host
     installer.exec_command(sudo=True, cmd=SSH_COPYID.format(key, user, node.hostname))
+
+
+def get_pid_limit(node, service):
+    """
+    Gets the pids limit of a given service
+    Args:
+        node (CephNode): Ceph node where the service is running
+        service (str): Name of the service (rgw, mon, mds)
+    """
+    # Identify the process name of the given service
+    cmd = f"podman ps | grep {service}"
```

Comment:
Add method for this step here - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L47